src/helm/benchmark/scenarios/ice_scenario.py

```diff
@@ -358,6 +359,12 @@ def filter_by_metadata(self, subset: ICESubset, header_dir: str, all_corpus_file
                 return []
 
             for fi, columns in files:
+
+                try:
+                    import xlrd  # noqa
```

Comment:
Why not do this at the top-level like the other imports of optional dependencies?